modules/internal/ChapelIO.chpl

```diff
@@ -403,23 +582,48 @@ module ChapelIO {
           }
         } else {
           // Read the field name = part until we get one that worked.
+          var found_field = false;
           for param i in 1..num_fields {
             var st = reader.styleElement(QIO_STYLE_ELEMENT_AGGREGATE);
             var eq:ioLiteral;
```

Comment:
It looks like you've left "eq" in this scope here but you redeclare it in an inner scope.  I'm guessing that wasn't intentional?